lms/djangoapps/instructor/views/registration_codes.py

```diff
@@ -0,0 +1,117 @@
+"""
```

Comment:
This is essentially an API. I highly recommend using [Django REST Framework](http://www.django-rest-framework.org/) to implement these views.